decontamination/README.md

```diff
@@ -22,3 +22,23 @@ The results from a previous decontamination run can be used to speed-up the scri
 ```bash
 python find_substrings.py --dataset-name bigcode/stack-dedup-alt-filter-no-pii --output-dir /path/to/output  --num-proc 32 --cached-decontamination-dir /path/to/previous/output/ --cache-retrieval-key content --split-languages
 ```
+
+## Near Matching with MinHash and LSH
+
+This is similar to the near deduplication script `data_analysis/near-deduplication/minhash_deduplication_alt.py` with one modification: we use benchmark datasets as index source instead of the dataset itself.
+
+### Usage:
+1. Update the script to include any benchmark you want to check against in `DATASETS_TO_CHECK`. Be sure to create a global variable for the index using the same name in that config. Benchmark columns should be of type string or sequence of string, so that they can be concatenated.
```

Comment:
```suggestion
1. Update the script to include any benchmark (from Hugging Face hub) you want to check against in `DATASETS_TO_CHECK` variable from `minhash.py`. Be sure to create a global variable for the index using the same name in that config. Benchmark columns should be of type string or sequence of string, so that they can be concatenated.
```